assets/src/application/ui/layout/entityList/filterBar/buttons/TableView.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import classNames from 'classnames';
+import { __ } from '@wordpress/i18n';
+
+import { EspressoButton, EspressoButtonType, Icon } from '@application/ui/input';
+import FilterButtonWrap from './FilterButtonWrap';
+import { TableViewFilterButtonProps } from '../types';
+
+const TableViewFilterButton: React.FC<TableViewFilterButtonProps> = ({ listId, setTableView, view }) =>
+	React.useMemo(() => {
+		const className = classNames('ee-entity-list-filter-bar__btn', { 'ee-active-filters': view === 'table' });
```

Comment:
ay some of these existing selector names are not very BEM friendly eh?

I think any refactoring pertaining to that is beyond the scope of this PR however and should therefore be done in another one.

Reply:
I've renamed `ee-entity-list-filter-bar` to `ee-filter-bar` and added modifiers